src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -110,6 +110,11 @@ private ISymbolNode CreateReadyToRunHelperCell(ReadyToRunHelper helperId)
             SignatureContext signatureContext, 
             bool isUnboxingStub = false)
         {
+            if (targetMethod.HasCustomAttribute("System.Runtime.CompilerServices", "IntrinsicAttribute"))
```

Comment:
This check is too pessimistic. There are many intrinsics that can be AOT compiled just fine.

Reply:
crossgen checks for specific types like `Vector` that depend on the current processor.

Reply:
In the 2nd iteration I have added a specific check for the Vector`1 type like Crossgen does.